### PR TITLE
background - persistence pipeline - fix persistence bug

### PR DESCRIPTION
**Pull request by kumavis**

### CODE REVIEW

File: app/scripts/background.js [hunk 1]
```diff
@@ -273,7 +274,7 @@ function setupController (initState, initLangCode) {
     asStream(controller.store),
     debounce(1000),
     storeTransform(versionifyData),
-    storeTransform(persistData),
```
Comment (danfinlay) [inline app/scripts/background.js]: So a transform stream will just stop if it doesn't get read from?

**Review by danfinlay — DISMISSED** [review]
Definitely deserves an exciting changelog entry!

File: app/scripts/background.js (hunk 1 above)
Comment (kumavis) [reply app/scripts/background.js]: it has a `highWatermark` option which defaults to 16

**Review by danfinlay — APPROVED** [review]
Looks good, thank you!!!

**Pull request merged**